### PR TITLE
Minimal Changes to remove Python2.7 Cruft

### PR DESCRIPTION
**Pull request by jamesba**
No type checking or anything like that yet, but I've removed the cruft needed for dual stacking.

### CODE REVIEW

**Comment by jamesba** [comment]
Job: https://www.pivotaltracker.com/story/show/169124739

File: Makefile [hunk 1]
```diff
@@ -1,9 +1,8 @@
 PYTHON=`which python`
-PYTHON2=`which python2`
 PYTHON3=`which python3`
```
Comment (philipnbbc) [inline Makefile]: Should `PYTHON` be removed as well and replaced by `PYTHON3`? Or is there a reason to use the default python version to build the source and install targets?

**Comment by philipnbbc** [comment]
Would it be useful as part of this PR to add a Makefile `lint` target that runs `flake8` via tox?

File: Makefile (hunk 1 above)
Comment (philipnbbc) [reply Makefile]: Possibly related: all the source files have a `#!/usr/bin/python` at the top. Should these be changed to refer to python3? Or maybe they should be removed?

File: mediagrains/gsf.py [hunk 2]
```diff
@@ -704,16 +702,8 @@ def _write_rational(file, value):
...
-        return file.seekable()
-    else:
-        try:
-            file.tell()
-        except IOError:
-            return False
-        else:
-            return True
+def seekable(file):
+    return file.seekable()
```
Comment (philipnbbc) [inline mediagrains/gsf.py]: Minor: maybe worth removing this function and replacing calls with `file.seekable`?

**Pull request merged**